### DISCUSSION TITLE
How to get the functionality of input mouse Drag to change value

### DISCUSSION THREAD

**Discussion by vamshi0** [GrapesJS/grapesjs#4552] [Q&A]
Hi all,

i am trying to have a mousedown & drag event on a seperate div element to change the input value of properties,

but, i am unable to find the function/command that will trigger the functionality,

any help is appreciated, Thank you..!

<img width="140" alt="Screenshot 2022-09-05 103152" src="https://user-images.githubusercontent.com/19800624/188364648-a222a4ac-2c60-4e50-b9ee-2be29e132a78.png">

**Comment by artf** [comment]
I'd recommend checking the docs for [Style Manager UI customization](https://grapesjs.com/docs/modules/Style-manager.html#customization)
